en-US/firefox-ios.xliff

```diff
@@ -1812,16 +1819,6 @@
         <target>Reopen Closed Tab</target>
         <note>Hardware shortcut to reopen the last closed tab, from the tab or the tab tray. Shown in the Discoverability overlay when the hardware Command Key is held down.</note>
       </trans-unit>
-      <trans-unit id="Request Desktop Site">
```

Comment:
Looks like these are old strings replaced with `Menu.ViewDekstopSiteAction.Title` and `Menu.ViewMobileSiteAction.Title`.